tgstation.dme

```diff
@@ -216,6 +261,7 @@
 #include "code\datums\helper_datums\teleport.dm"
 #include "code\datums\helper_datums\topic_input.dm"
 #include "code\datums\spells\area_teleport.dm"
+#include "code\datums\spells\barnyard.dm"

```

Comment:
Alright thanks
I'm going to make a poll for it anyway and if people prefer this I will just edit the horseman curse spell anyway or delete this.
I will keep that in mind if I ever have to add a file in the future.